hknweb/events/models.py

```diff
@@ -22,8 +24,16 @@ class Event(models.Model):
     end_time    = models.DateTimeField()
     location    = models.CharField(max_length=255)
     event_type  = models.ForeignKey(EventType, models.CASCADE)
-    description = models.TextField()
+    description = MarkdownxField()
     rsvp_limit  = models.PositiveIntegerField(null=True, blank=True)
+    access_level = models.IntegerField(
```

Comment:
Is there a level for officers/should there be?

Reply:
I originally thought there should be a distinction between officers and members, but then I was wondering how often (if at all) we would ever need that distinction (like are there any officer events that members shouldn't see?). What do you think?

Reply:
Closing for now! (we'll probably end up doing access control in a less terrible way later on too lmao)

Reply:
Yeah this makes a lot of sense (sorry this slipped by).